wasm/jsapi/memory/grow.any.js

```diff
@@ -1,10 +1,9 @@
-// META: global=jsshell
+// META: global=default,jsshell
```

Comment:
Note that this is not a functional change. I only did it for clarity. See 23111.

Reply:
Please revert this change, we can make it in #23111.